addons/binding/org.openhab.binding.yeelight/META-INF/MANIFEST.MF

```diff
@@ -20,7 +18,19 @@ Import-Package:
  org.eclipse.smarthome.core.types,
  org.openhab.binding.yeelight,
  org.openhab.binding.yeelight.handler,
+ org.openhab.binding.yeelight.lib,
+ org.openhab.binding.yeelight.lib.device,
+ org.openhab.binding.yeelight.lib.device.connection,
+ org.openhab.binding.yeelight.lib.enums,
+ org.openhab.binding.yeelight.lib.listeners,
+ org.openhab.binding.yeelight.lib.services,
  org.slf4j
 Service-Component: OSGI-INF/*.xml
 Export-Package: org.openhab.binding.yeelight,
- org.openhab.binding.yeelight.handler
+ org.openhab.binding.yeelight.handler,
+ org.openhab.binding.yeelight.lib,
```

Comment:
As the yeelight lib packages haven't been exposed before, there is probably no need to do so now.
Could you rename the package to `org.openhab.binding.yeelight.internal.lib` and remove these exports?